src/views/Restoration.vue

```diff
@@ -225,10 +231,17 @@ export default class Restoration extends Vue {
       // set Applicant info in entitySnapshot
       const stateFiling = entitySnapshot.businessInfo.stateFiling
       const filing = stateFiling && await LegalServices.fetchFiling(stateFiling)
-      entitySnapshot.orgPersons = filing.restoration.parties as OrgPersonIF
-      entitySnapshot.orgPersons.forEach(o => {
-        o.officer.id = uuidv4()
-      })
+      if (!filing) {
+        throw new Error(`Invalid fetched stateFiling = ${this.getBusinessId}`)
+      } else {
+        const parties = filing.restoration.parties
```

Comment:
`restoration` could still be undefined or null. Try this:
`const parties = filing.restoration?.parties || []`

Reply:
If no parties were provided, does this filing fail gracefully? Specifically, can the user enter a new applicant if there is no previous one? (You could temporarily hack this code to test it locally.)